docs/pages/desktop-access/getting-started.mdx

```diff
@@ -1,213 +1,134 @@
 ---
 title: Getting Started with Desktop Access
-description: Automatically add Teleport Desktop Access to your infrastructure
+description: Access Windows systems through Teleport Desktop Access
 ---
 
```

Comment:
How about a note at the top to help orient the reader and ensure they're on the right page? Something like:

```
Using Active Directory?

This guide sets up Teleport Desktop Access for standalone hosts that are not
joined to an Active Directory domain. To integrate Teleport with AD, see our
Active Directory guide.
```

Reply:
Yes, I still need to write the intro to the doc. This is a draft PR, staged as such while I make sure the technical details are correct as they are confirmed.